pallets/gear-debug/src/tests.rs

```diff
@@ -436,6 +449,13 @@ fn check_not_allocated_pages() {
         persistent_pages.insert(gear_page0, page0_data.to_vec());
         persistent_pages.insert(gear_page7, page7_data.to_vec());
 
+        // For all pages, which is write accessed, and has no data in storage yet,
+        // we must upload to storage all pages from [PAGE_STORAGE_GRANULARITY] interval.
+        [gear_page0, gear_page7]
+            .iter()
```

Comment:
```suggestion
            .into_iter()
```
And remove `copied()`.
Such method can be used since 2021 edition

Reply:
I dont actually understand the reason, but warnings appear then:
```
this method call resolves to `<&[T; N] as IntoIterator>::into_iter` (due to backwards compatibility), but will resolve to <[T; N] as IntoIterator>::into_iter in Rust 2021
`#[warn(array_into_iter)]` on by default
this changes meaning in Rust 2021
for more information, see <https://doc.rust-lang.org/nightly/edition-guide/rust-2021/IntoIterator-for-arrays.html>rustc(array_into_iter)
tests.rs(455, 14): use `.iter()` instead of `.into_iter()` to avoid ambiguity: `iter`
tests.rs(454, 9): or use `IntoIterator::into_iter(..)` instead of `.into_iter()` to explicitly iterate by value: `IntoIterator::into_iter(`, `)`
```

Reply:
But I will remove copied and make |&page | {}

Reply:
Done